src/provider/shplonk.rs

```diff
@@ -228,9 +224,9 @@ where
   fn verify(
     vk: &KZGVerifierKey<E>,
     transcript: &mut <NE as NovaEngine>::TE,
-    _C: &Commitment<NE>,
+    C: &Commitment<NE>,
     point: &[E::Fr],
-    _P_of_x: &E::Fr,
+    P_of_x: &E::Fr,
     pi: &EvaluationArgument<E>,
   ) -> Result<(), NovaError> {
     let r = HyperKZG::<E, NE>::compute_challenge(&pi.comms, transcript);
```

Comment:
`C` (which should be equal to `pi.comms[0]` but should be removed to avoid the duplication) does not need to be absorbed here since it is assumed to be part of the transcript already.